advanced_samples/CSV/dotnet/CSVtoOCS_pkce/CSVtoOCS_pkce.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
In the polaris branch I've been adding a `Solution Items` folder to the `.sln` with:

- `azure-pipeline.yml`
- `Directory.build.props` (from root level)
- `Extensions.ruleset` (from root level)
- `HISTORY.md`
- `OSIsoft.ruleset` (from root level)
- `polaris.yml` (obviously in my branch only)
- `README.md`

Reply:
add all i can right now